app/examples/modal.tsx

```diff
@@ -30,6 +30,7 @@ const ModalScreen = () => {
           title="Confirm modal"
           text="Optional text"
           primaryActionLabel="Some action"
+          // eslint-disable-next-line no-console
```

Comment:
we are using the console logs in so many places and do not disable the warning

Reply:
Yes, we should disable the warning on on those places where console log is intentional and permanent.
When console log is just temporary, we should keep the warning on, for future reference.